jsapp/js/components/processing/sidebar/sidebarDisplaySettings.tsx

```diff
@@ -32,14 +32,17 @@ export default function SidebarDisplaySettings() {
   const availableDisplays = store.getAvailableDisplays(activeTab);
```

Comment:
I like that we are making sure that the `getStaticDisplayText` returns data that makes sense. But I think that `getAvailableDisplays` should too. So if there is no transcript, it shouldn't propose it (i.e. shouldn't `outcome.push(StaticDisplays.Transcript);`)

Reply:
If we fix `getAvailableDisplays`, then I think we don't need all that:

```
                const staticDisplayText = getStaticDisplayText(staticDisplay);
                if (staticDisplayText === null) {
                  return null;
                }
```